lib/main.dart

```diff
@@ -68,13 +68,11 @@ class CircuitVerseMobile extends StatelessWidget {
             builder: (themeContext) => GetMaterialApp(
               title: 'CircuitVerse Mobile',
               localizationsDelegates: [
-                AppLocalizationsDelegate(),
+                AppLocalizations.delegate,
                 GlobalMaterialLocalizations.delegate,
                 GlobalWidgetsLocalizations.delegate,
               ],
```

Comment:
If we are using generated `supportedLocales` then we should also be using generated delegates

localizationsDelegates: AppLocalizations.localizationsDelegates,

Reply:
using generated delegates instead of defining them explicitly!